### DISCUSSION TITLE
Switching Kubernetes Versions in the all.yml File?

### DISCUSSION THREAD

**Discussion by epoch-philosophy** [techno-tim/k3s-ansible#419] [Questions] [answered]
Hi, I am very curious as to whether switching Kubernetes v1.25, to say, v1.29 would cause problems? Or why there is a reason this is specific playbook is running a release from early 2023?

Curious to see why this may be the case!

**Comment by timothystewart6** [comment, marked as answer]
Hey!  Go for it!  Just set the version and give it a shot!  We will be upgrading soon as soon as our CI jobs catch up to our PRs!

**Reply by epoch-philosophy** [reply]
Awesome. Thanks for the reply and all the wonderful work.

So, got it up and running with the latest K3S version (v1.29), with latest KubeVIP and MetalLB. Nothing performance or stability wise seems to be hampered. That said, I don't think Rancher supports K3S above 1.27, so that kinda sucked lol. That said, Kubectl with Rook should be perfectly fine for me.

**Reply by timothystewart6** [reply]
Nice!  Yeah, that's why I have stayed on 1.26 for so long.  Waiting for rancher to be updated.  I don't need the latest features, I am fine with a stable version that's still supported!
